backend/utils/db.py

```diff
@@ -79,10 +84,11 @@ def update_snack(sku: str, updates: SnackUpdateSchema) -> Snack:
         cursor = conn.cursor()
         cursor.execute("""
             UPDATE snacks 
-            SET name = ?, quantity = ?
+            SET name = ?, quantity = ?, price = ?, 
+                description = ?, category = ?, photo_url = ?           
             WHERE sku = ?
             RETURNING *
-        """, (updates.name, updates.quantity, sku))
+        """, (updates.name, updates.quantity, updates.price, updates.description, updates.category, updates.photo_url, sku))
```

Comment:
minor formatting suggestion
```suggestion
        """, (
            updates.name, 
            updates.quantity, 
            updates.price, 
            updates.description, 
            updates.category, 
            updates.photo_url, 
            sku
        ))